go/libkb/db.go

```diff
@@ -20,6 +20,7 @@ const (
 	DBTeamChain         = 0x10
 	DBUserPlusAllKeysV1 = 0x19
 
+	DBContactResolution              = 0xb0
```

Comment:
skipped 0xba?

Reply:
I didn't know which one to use, they've all been pretty arbitrary

Reply:
Just decrement by 1, I think the times we've skipped have been errors. Also might want to think about making this key 2 bytes soon

Reply:
I remember now. i put it temporarily here and I was going to stuck it somewhere in the middle where there is space, like 0xc9. I don't think that matters a lot, though,  I'm changing it to 0xba.